src/proto/thread_telemetry.proto

```diff
@@ -14,20 +14,19 @@ package threadnetwork;
 // Usage:
 // Delete field: do not directly delete field. Deprecate it instead.
 message TelemetryData {
-  enum NodeType {
-    NODE_TYPE_UNSPECIFIED = 0;
-    NODE_TYPE_ROUTER = 1;
-    NODE_TYPE_END = 2;
-    NODE_TYPE_SLEEPY_END = 3;
-    NODE_TYPE_MINIMAL_END = 4;
+  message Duration {
```

Comment:
nit: include license header on this file?

Reply:
Done.